sdks/versions.mk

```diff
@@ -23,7 +23,9 @@ ANDROID_SDK_VERSION_x86_64?=21
 
 # iOS
 
-XCODE_DIR?=/Applications/Xcode.app/Contents/Developer
+XCODE_DIR?=/Applications/Xcode101.app/Contents/Developer
+# Xcode version used to compile 32 bit cross compilers
+XCODE32_DIR?=/Applications/Xcode94.app/Contents/Developer
```

Comment:
This maps to the configuration we have on our CI machines: https://github.com/mono/mono/pull/11691#issuecomment-439178459